front-end/src/components/List/ListItem/ReaderListItem.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+import styles from './ReaderListItem.scss';
+
+interface OwnProps {
+  id: string;
+  name: string;
```

Comment:
```suggestion
  firstname: string;
```
This would be nicer.